### ISSUE TITLE
Brave browser doesn't connect to ZeroNet

### ISSUE THREAD

**Issue by dmp1ce** [HelloZeroNet/ZeroNet#612]
When I try to connect to ZeroHello using the Brave browser I don't see any content and the message `Connection with UiServer Websocket was lost. Reconnecting...` keeps reappearing.

**Comment by unsystemizer** [comment]
That's false. It's easy to see on the ZeroNet server that connections to port 43110 are actually established.
And it actually works.

![image](https://cloud.githubusercontent.com/assets/13134193/19437338/8ca44114-94a7-11e6-8109-af0226814fc4.png)

**Comment by dmp1ce** [comment]
@unsystemizer Thanks for the update. I am actually having the issue in MultiUser mode. By default, Zeronet *does* work with Brave.

**Comment by unsystemizer** [comment]
Okay. Can you paste the startup command(s) so that people can duplicate the issue?

**Comment by dmp1ce** [comment]
I just running a normal install but turning on MultiUser. The easiest way to test is to try a proxy site like: https://onlyzero.net/

I have the issue on that proxy with the Brave browser.

**Comment by HelloZeroNet** [comment]
I have downloaded the browser and can confirm it's not working properly. The only solution I have found is setting cookie controll to allow third part on page about:preferences#shields.

It's a known problem: https://github.com/brave/browser-laptop/issues/2417 I will try to find a workaround later.

**Comment by dmp1ce** [comment]
Probably something that should be fixed in Brave anyway.

**Comment by unsystemizer** [comment]
Allow 3rd party cookie control is required on single instance as well, it's not something specifically related to multi-instance
